gradle.properties

```diff
@@ -189,5 +189,29 @@ jfigletVersion=0.0.8
 
 embeddedRedisVersion=0.6
 
+#webjars versions
```

Comment:
I still haven't tested things in the webapp, but now the unit tests are passing, and I have the versions being filtered into static-uris.properties. I moved static-uris.properties to the same jar where it is used. Someone could always override it by putting a copy in their overlay (web-inf/classes/static-uris.properties would override version in jar) but that isn't really something anyone would want to do. They could also override individual values in static-uris.properties by putting the key in custom_messages.properties (also not something I see a reason anyone would do).

Regarding testing, I was planning to test the overlay once 5.1.0 was released so I could try out the new overlay. How are you planning to deal with 5.1.0 in the overlay project? Will the changes go in to master or in a 5.1 branch?